eventlet/hubs/asyncio.py

```diff
@@ -0,0 +1,120 @@
+"""
+Asyncio-based hub, originally implemented by Miguel Grinberg.
+"""
+
+import asyncio
+import os
+import sys
+
+from eventlet.hubs import hub
+
+
+def is_available():
+    """Python 3 always has asyncio."""
+    return True
+
+
+class Hub(hub.BaseHub):
+    """An Eventlet hub implementation on top of an asyncio event loop."""
+
+    @staticmethod
+    def is_available():
```

Comment:
Same thing here. I'm not sure what kind of availability is expected here, but I think this is related to python releases, so if I'm right, this part could be also removed.

Reply:
This can probably be removed.